src/Compute/Compute.Autorest/docs/Remove-AzGalleryApplicationVersion.md

```diff
@@ -32,8 +32,7 @@ Delete a gallery Application Version.
 
 ### Example 1: Remove a Gallery Application Version 
 ```powershell
-PS C:\> Remove-AzGalleryApplication -ResourceGroupName $rgNmae -GalleryName $galleryName -GalleryApplicationName $galleryApplicationName -Name $name
-
+Remove-AzGalleryApplication -ResourceGroupName $rgNmae -GalleryName $galleryName -GalleryApplicationName $galleryApplicationName -Name $name
```

Comment:
Here should be the example of Remove-AzGalleryApplicationVersion, need to change Remove-AzGalleryApplication to Remove-AzGalleryApplicationVersion